src/acsets/amr_conversion/stockflow.py

```diff
@@ -0,0 +1,187 @@
+"""
+This module contains two methods used for converting between stock and flow
+acsets and amr. Both methods accept an acset/amr JSON dictionary object and
+returns its counterpart JSON dictionary object.
+"""
+import requests
+import sympy
+import re
+
+
+def acset_to_amr(acset):
+    """
+    A method that takes in a stock and flow JSON acset dictionary object
+    and outputs an equivalent stock and flow JSON amr dictinary object
+    """
+    stocks = acset['Stock']
+    flows = acset['Flow']
+
+    stocks_list = []
+    flows_list = []
+    links_list = []
+    params_list = []
+    initials_list = []
+    auxiliaries_list = []
+
+    params = []
+    params_stock_flow_map = {}
+
+    for stock in stocks:
+        stocks_list.append({'id': stock['sname']})
+        initials_list.append({'target': stock['sname']})
+
+    for idx, flow in enumerate(flows):
+        flow_id = 'flow' + str(flow['_id'])
+        flow_name = flow['fname']
+        upstream_stock = stocks[int(flow['u']) - 1]['sname']
+        downstream_stock = stocks[int(flow['d']) - 1]['sname']
+
+        expression_str = flow['ϕf'].replace('p.', '').replace('u.', '')
+
+        flow_dict = {}
+        flow_dict['id'] = flow_id
+        flow_dict['name'] = flow_name
+        flow_dict['upstream_stock'] = upstream_stock
+        flow_dict['downstream_stock'] = downstream_stock
+        flow_dict['rate_expression'] = expression_str
+
+        params.extend(re.findall(r'p\.([^()*+-/ ]+)', flow['ϕf']))
+
+        params_stock_flow_map[flow_id] = []
+
+        params_stock_flow_map[flow_id].extend(
+            re.findall(r'p\.([^()*+-/ ]+)', flow['ϕf']))
```

Comment:
doesn't this assume that a parameter always starts with `p.`?

Reply:
Yes, we assumed that was the case.